forge-gui/res/cardsfolder/upcoming/dont_blink.txt

```diff
@@ -0,0 +1,8 @@
+Name:Don't Blink
+ManaCost:1 U
+Types:Instant
+A:SP$ Effect | ReplacementEffects$ ReplaceETB | SpellDescription$ Until end of turn, if one or more creatures would enter the battlefield from exile or after being cast from exile, their owners shuffle them into their libraries instead.
+SVar:ReplaceETB:Event$ Moved | Destination$ Battlefield | Origin$ Stack,Exile | ValidCard$ Creature.wasCastFromExile,Creature.wasNotCast | ReplaceWith$ DBShuffle | Description$ If CARDNAME would be put into a graveyard from anywhere, reveal CARDNAME and shuffle it into its owner's library instead.
+SVar:DBShuffle:DB$ ChangeZone | Hidden$ True | Origin$ All | Destination$ Library | Defined$ ReplacedCard | Shuffle$ True
```

Comment:
@Hanmac 
you think we need followup issue for this - 
like it should probably only be a single Shuffled trigger vs. something like _Helvault_?

Reply:
hmm, seems likely to me

are there rules updates for this stuff?

Reply:
probably needs to be a single shuffle effect